bakery-src/tests/test_bakery_scripts.py

```diff
@@ -1154,11 +1155,11 @@ def test_check_feed(tmp_path, mocker):
             "consumer": "REX",
             "commit_sha": "ffffffffffffffffffffffffffffffffffffffff",
         },
-        # It should not try to queue this version: repo and commit match prev
+        # It should NOT try to queue this version: repo and commit match prev
         {
             "repository_name": "osbooks-introduction-sociology",
             "code_version": "20210224.204120",
-            "uuid": "02040312-72c8-441e-a685-20e9333f3e1e",
```

Comment:
Does this change mean anything? I couldn't seem to load the book with either of these uuids

Reply:
Oh yeah, sorry. I made this change purely to make the data more reflective of real data. It does not actually matter at all. I just got a little confused when looking at the data and seeing the difference. It made me think that it was important that the uuid was different. In reality, it's only the repository name and commit sha that matter for check feed. That's also why I changed not to NOT.